src/assets/styles/global/typography.scss

```diff
@@ -0,0 +1,28 @@
+.heading,
+%heading {
+  font-family: $font-heading;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  @extend %heading;
+}
+
+body {
+  font-family: $font-body;
+  font-weight: $font-body-weight;
+  font-style: $font-body-style;
+}
+
+p,
+%p {
+  font-family: $font-body;
+}
+
+ul {
+  @extend %p;
```

Comment:
For readability, what do you like : the following code or the current one?

`
p, ul {
  @extend %p;
}
`

Because I notice sometimes you enumerate your selectors (e.g. `input, textarea, select`) to style and sometimes it's `@extend` placeholders so I was wondering which method is preferable.

While I like Sass placeholders, I remember it doesn't like media queries when compiling CSS so if we add support with responsive, maybe we can use other solutions but so far it's good!